### PR TITLE
Highlighting the difference between client node and training client.

### PR DESCRIPTION
**Pull request by sztoor**

### CODE REVIEW

File: docs/quickstart.rst [hunk 1]
```diff
@@ -19,6 +19,11 @@ Start by creating an account in FEDn Studio and set up a project by following th
 Install FEDn
 ------------
 
+Client node: A physical or virtual machine (MacOSx Linux or Windows) used to perform initial steps.
```
Comment (Wrede) [inline docs/quickstart.rst:22]: this just makes it more complex for the user I think, Also, it sound to much "hadoop"

File: docs/quickstart.rst [hunk 2]
```diff
@@ -41,17 +46,18 @@ It is recommended to use a virtual environment when installing FEDn.
 
 .. _package-creation:
 
-Initialize FEDn with the client code bundle and seed model 
+Initialize FEDn with the compute package and seed model
```
Comment (Wrede) [inline docs/quickstart.rst:49]: We want to move away from calling it compute package and instead refer to is as a FEDn project which can be bundled into a package which then can be used by the fedn package manager.

File: docs/quickstart.rst [hunk 3]
```diff
@@ -41,17 +46,18 @@ It is recommended to use a virtual environment when installing FEDn.
...
 
-Initialize FEDn with the client code bundle and seed model 
+Initialize FEDn with the compute package and seed model 
 ----------------------------------------------------------
 
-Next, we will prepare the client. The key part of a FEDn Project is the client definition - 
+Next, we will prepare the training client. The key part of a FEDn Project is the client definition - 
 code that contains entrypoints for training and (optionally) validating a model update on the client. 
 
+[!NOTE] The preparation of the compute package and seed model is a one-time requirement. All training clients use the same compute package and seed model.
```
Comment (Wrede) [inline docs/quickstart.rst:55]: remove compute

File: docs/quickstart.rst [hunk 4]
```diff
@@ -41,17 +46,18 @@ It is recommended to use a virtual environment when installing FEDn.
...
 code that contains entrypoints for training and (optionally) validating a model update on the client. 
 
+[!NOTE] The preparation of the compute package and seed model is a one-time requirement. All training clients use the same compute package and seed model. 
+
 Locate into ``examples/mnist-pytorch`` and familiarize yourself with the project structure. The dependencies needed in the client environment are specified 
 in ``client/python_env.yaml``. 
 
-In order to train a federated model using FEDn, your Studio project needs to be initialized with a compute package and a seed model. The compute package is a bundle
-of the client specification, and the seed model is a first version of the global model.  
+In order to train a federated model using FEDn, your Studio project needs to be initialized with a compute package and a seed model. The compute package is a bundle of the client specification, and the seed model is a first version of the global model.
```
Comment (Wrede) [inline docs/quickstart.rst:60]: remove compute

**Review by Wrede — CHANGES_REQUESTED** [review]

**Comment by ahellander** [comment]
I think this has been superseeded with some of our more recent updates.

**Pull request closed without merging**